caos.ansible_roles/roles/assert_no_service/tasks/assert-service-stopped-Linux.yaml

```diff
@@ -0,0 +1,11 @@
+---
+
+# FROM package_uninstall
+- name: Populate service facts
+  ansible.builtin.service_facts:
+
+- name: ensure infra-agent service is not present
+  assert:
+    that: ansible_facts.services[service_name] is not defined
```

Comment:
will service_name be expanded? should have the variable format?

Reply:
🕵️‍♂️ nice catch! (and shame to my tests :( ) fixed in 41f623004392499de96242286a11b0ec4fd468a0